templates/buyer/product/show.html.twig

```diff
@@ -137,4 +155,21 @@
             $('#feedbacks').DataTable();
         } );
     </script>
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    <style>
```

Comment:
please create a product.css and move all styling related to product to that file. include that file where needed. this is not correct.

Reply:
all files which are placed in public should not be there. please move them in `assets`. apart from that use of cdn files as links should be kept to minimum as much as possible. for example datatables file link in layout shouldn't  be use that way. instead either a package manager should be used or at least download the file and use it from assets.

Reply:
Done